mycli/packages/expanded.py

```diff
@@ -17,7 +28,8 @@ def expanded_table(rows, headers):
     header_len += 2
 
     for row in rows:
-        row_len = max([len(_text_type(x)) for x in row])
+        row = [format_field(x) for x in row]
+        row_len = max([len(x) for x in row])
```

Comment:
Can this be collapsed into a single line? 

```
row_len = max([len(format_field(x)) for x in row])
```

That way we don't keep two lists around just to find out the max length of the row.